conf/broker.conf

```diff
@@ -1203,6 +1203,13 @@ brokerServicePurgeInactiveFrequencyInSeconds=60
 transactionCoordinatorEnabled=false
 transactionMetadataStoreProviderClassName=org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider
 
+# Transaction buffer take snapshot interval number
+transactionBufferTakeSnapshotIntervalNumber=1000
```

Comment:
The number is the transactions count or entry count?  And it's better to use `max` as the prefix?